src/ScanPatron/ScanForm.test.js

```diff
@@ -163,4 +160,23 @@ describe('ScanForm', () => {
       expect(screen.getAllByText("ui-reading-room.userDoesn'tExist")).toBeDefined();
     });
   });
+
+  describe('NotesSmartAccordion', () => {
+    it('should be called with correct props', () => {
```

Comment:
I don't really think that UI testing means checking a component's props, but rather what a user can actually see on UI. It is also written in react-testing-library [doc](https://testing-library.com/docs/#what-you-should-avoid-with-testing-library)